src/status.md

```diff
@@ -129,7 +129,7 @@ As such, certain higher-level language features still have their limitations or
 | Union&nbsp;types                                  | Union types are not supported yet, except for nullable class types. There is no `any` type. A viable alternative is to use generics specialized with [static type checks](./stdlib/globals.md#static-type-checks) to achieve a similar effect.
 | Symbols                                           | Symbols are implemented in the standard library, but don't have deep compiler integration yet.
 | Object literals                                   | Object literals can be used in places where the current type is a bare class, then corresponding to an instantiation of the class.
-| JSON                                              | JSON is not strictly typed in nature, so we haven't settled on a standard yet. Solutions developed by the community: [assemblyscript-json](https://github.com/nearprotocol/assemblyscript-json)
+| JSON                                              | JSON is not strictly typed in nature, so we haven't settled on a standard yet. Solutions developed by the community: [json-as](https://github.com/JairusSW/json-as), [assemblyscript-json](https://github.com/nearprotocol/assemblyscript-json)
```

Comment:
```suggestion
| JSON                                              | JSON is not strictly typed in nature, so we haven't settled on a standard yet. Solutions developed by the community: [json-as](https://github.com/JairusSW/json-as)
```
I'd even go so far as to remove assemblyscript-json, because it's unmaintained and apparently also very slow and memory-intensive, and none of those issues can be said for json-as.